charts/dgraph/example_values/ingress/ingress-nginx-grpc.yaml

```diff
@@ -0,0 +1,43 @@
+# ingress-nginx with GRPC support
+# --------------------------------------
+# TLS will be terminated by the ingress controller.  The ingress controller
+# should be configured an Layer 4 external load balancer.
+# Two ingress resources will configured for GRPC support: GPRC and HTTPS.
+# References:
+# * https://github.com/kubernetes-sigs/external-dns/
+# * https://cert-manager.io/
+# * https://kubernetes.github.io/ingress-nginx/
+# * https://kubernetes.github.io/ingress-nginx/examples/grpc/
+# * https://github.com/kubernetes/ingress-nginx/tree/master/docs/examples/grpc
+global:
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: nginx
+      nginx.ingress.kubernetes.io/ssl-redirect: "true"
+      nginx.ingress.kubernetes.io/backend-protocol: HTTP
+      cert-manager.io/cluster-issuer: letsencrypt-prod
+    tls:
+    # All hostnames must be listed for grpc + https ingress shared cert secret
+    - hosts:
+      - ratel.example.com
+      - alpha.example.com
+      - grpc.example.com
```

Comment:
Is this ingress for HTTP? This looks like the HTTP section but I see this example for `grpc.example.com`.